astropy/units/physical.py

```diff
@@ -542,9 +544,21 @@ def get_physical_type(obj):
         return PhysicalType(unit, "unknown")
 
 
+# ------------------------------------------------------------------------------
+# Script section creating the physical types and the documentation
+
+# define the physical types
 for unit, physical_type in _units_and_physical_types:
     def_physical_type(unit, physical_type)
 
+# key-view of physical type names
```

Comment:
I'm not sure I see a point to this. I think it makes more sense to just expose the dictionary itself (and call it `physical_types`). However, even better may be to do this via an interface (e.g., on the unit registry), and perhaps this is an argument to postpone this until we have thought a bit about the interface.